package.json

```diff
@@ -77,6 +77,7 @@
     "npm": "8.19.2"
   },
   "dependencies": {
-    "@storybook/addon-toolbars": "^6.5.16"
+    "@storybook/addon-toolbars": "^6.5.16",
+    "react-error-boundary": "^4.0.4"
```

Comment:
Looks like this should be in `tooling/template-base` instead of root.

Will need to make the same changes over in the `hello-world` theme as well.